linkcheck/migrations/0005_alter_link_id_alter_url_id.py

```diff
@@ -1,27 +0,0 @@
-# Generated by Django 4.1.3 on 2022-11-24 15:58
-
-from django.db import migrations, models
-
-
-class Migration(migrations.Migration):
-
-    dependencies = [
-        ("linkcheck", "0004_remove_url_still_exists"),
-    ]
-
-    operations = [
-        migrations.AlterField(
-            model_name="link",
-            name="id",
-            field=models.BigAutoField(
-                auto_created=True, primary_key=True, serialize=False, verbose_name="ID"
-            ),
-        ),
-        migrations.AlterField(
-            model_name="url",
-            name="id",
-            field=models.BigAutoField(
-                auto_created=True, primary_key=True, serialize=False, verbose_name="ID"
-            ),
-        ),
-    ]
```

Comment:
Sorry, somehow during the rebase of the preview PR I missed this auto-generated migration :unamused: